test/e2e/utils/kubectl.go

```diff
@@ -90,3 +93,49 @@ func (k *Kubectl) Wait(inNamespace bool, cmdOptions ...string) (string, error) {
 	}
 	return k.Command(ops...)
 }
+
```

Comment:
The `kubectl.go` is a helper for the `kubectl` commands (kubectl apply/delete/get/logs). So, I do not think that we should move the specific KB commands (kubebuider <command>) which are helpers to tests the project to here. So, let's came back it for `test/e2e/utils/test_context.go`?